src/Control/Abstract/Modules.hs

```diff
@@ -157,7 +157,6 @@ data ResolutionError resume where
                 -> Language -- Language.
                 -> ResolutionError ModulePath
 
-  -- JZ:Shall it be FilePath? as this first value can be github.com/... for GoLang
   GoImportError :: FilePath -> ResolutionError [ModulePath]
```

Comment:
Perhaps it is better to write String here.  GoImportError can be followed with an uri like `https://github.com/....`, which it is not a file path.

Reply:
@zhujinxuan Yup, `String`, not `FilePath` or a typed path is correct here. Can you change it to `String` and add a comment explaining why this is so?